MatrixSDK/MXSession.h

```diff
@@ -1084,12 +1084,16 @@ typedef void (^MXOnBackgroundSyncFail)(NSError *error);
 /**
  Delegate for updating room summaries.
  By default, it is the one returned by [MXRoomSummaryUpdater roomSummaryUpdaterForSession:].
+ 
+ This property is strong as the only other reference to the delegate is a weak ref in the updaterPerSession table.
  */
 @property id<MXRoomSummaryUpdating> roomSummaryUpdateDelegate;
```

Comment:
I wonder if these two should really be called delegates though :)

Reply:
It's a very good point to be fair. I'm not sure it would be worth making source breaking changes to the SDK for this though. I doubt the account data updater is used much by others but the room summary updater could well be 😕